openstack/openstack_vm.yaml

```diff
@@ -99,29 +99,35 @@ node_types:
-              timeout:                  { type: integer, default: { get_property: [ SELF, timeout ] } } 
-              floating_ip_pools:        { type: list,    default: { get_property: [ SELF, floating_ip_pools ] } }
-              userdata:                 { type: string,  default: { get_property: [ SELF, userdata ] } } 
-              env:                      { type: map,     default: { get_property: [ SELF, env ] } } 
-              consul_server_address:    { type: string,  default: { get_property: [ SELF, consul_server_address ] } }
+              network:                  { type: string,  default: { get_property:  [ SELF, network               ] } } 
+              flavor:                   { type: string,  default: { get_property:  [ SELF, flavor                ] } } 
+              key_name:                 { type: string,  default: { get_property:  [ SELF, key_name              ] } } 
+              image:                    { type: string,  default: { get_property:  [ SELF, image                 ] } } 
+              security_groups:          { type: string,  default: { get_property:  [ SELF, security_groups       ] } } 
+              vm_name:                  { type: string,  default: { get_property:  [ SELF, name                  ] } } 
+              ansible_user:             { type: string,  default: { get_property:  [ SELF, username              ] } } 
+              ssh_config:               { type: string,  default: { get_property:  [ SELF, ssh_config            ] } }
+              include_exporter:         { type: boolean, default: { get_property:  [ SELF, include_exporter      ] } }
+              timeout:                  { type: integer, default: { get_property:  [ SELF, timeout               ] } } 
+              floating_ip_pools:        { type: list,    default: { get_property:  [ SELF, floating_ip_pools     ] } }
+              userdata:                 { type: string,  default: { get_property:  [ SELF, userdata              ] } } 
+              env:                      { type: map,     default: { get_property:  [ SELF, env                   ] } } 
+              consul_server_address:    { type: string,  default: { get_property:  [ SELF, consul_server_address ] } }
+              deployment_label:         { type: string,  default: { get_input:             deployment_label        } }
+              monitoring_id:            { type: string,  default: { get_input:             monitoring_id           } }
+              instance_name:            { type: string,  default: { get_attribute: [ SELF, tosca_name            ] } }
+              id:                       { type: string,  default: { get_attribute: [ SELF, id                    ] } } 
             implementation: 
               primary: playbooks/vm_create.yml 
               dependencies:
                 - playbooks/egi_refresh_token.yml
                 - artifacts/node_exporter.json.j2
```

Comment:
Missing dependency. node_exporter.service.j2 should also be declared as dependency, otherwise it wont be available inside playbook